cmd/git-sync/main.go

```diff
@@ -69,8 +69,12 @@ var flPassword = flag.String("password", envString("GIT_SYNC_PASSWORD", ""),
 
 var flSSH = flag.Bool("ssh", envBool("GIT_SYNC_SSH", false),
 	"use SSH for git operations")
+var flSSHSecret = flag.String("ssh-key", envString("GIT_SSH_KEY", "/etc/git-secret/ssh"),
+	"the ssh key to use")
```

Comment:
Can you make the comment here be something like 'the path to the SSH key to use" - specifically it is a path, not the key itself.

Reply:
or name it to ssh-key-file (to parallel ssh-known-hosts-file)

Reply:
or ssh-key-path, ssh-known-hosts-path and cookie-file-path